packages/client-search/src/types/Log.ts

```diff
@@ -68,4 +68,17 @@ export type Log = {
    * Index name of the log
    */
   readonly index: string;
+
+  /**
+   * Internal queries performed for this query.
+   */
+  readonly inner_queries: ReadonlyArray<{
+    /**
+     * Index of this query.
+     */
+    readonly indexName: string;
+    readonly queryID: number;
```

Comment:
My bad, the example I wrote in the issue is incorrect. `queryID` should be a string instead of `number.`

Reply:
```suggestion
    readonly queryID: string;
```